app/services/user-role-service.js

```diff
@@ -29,7 +30,7 @@ module.exports.addUserRole = function (userRole) {
 
 module.exports.getRole = function (role) {
   return getRole(role).then(function (result) {
-    return result
+    return result ?? { role: userRoles.STAFF }
```

Comment:
Nice - had to look this up as it appears to have landed since I last did any node
It returns the left hand side if it's not null, otherwise the right hand side
neater than 
`return result ? result : { role: userRoles.STAFF }`